cmd/snap-confine/mount-support.c

```diff
@@ -700,3 +700,44 @@ void sc_setup_user_mounts(struct sc_apparmor *apparmor, int snap_update_ns_fd,
 	sc_do_mount("none", "/", NULL, MS_REC | MS_SLAVE, NULL);
 	sc_call_snap_update_ns_as_user(snap_update_ns_fd, snap_name, apparmor);
 }
+
+void sc_ensure_snap_dir_shared_mounts(void)
+{
+	const char *dirs[] = { SNAP_MOUNT_DIR, "/var/snap", NULL };
+	for (const char **dir = dirs; *dir != NULL; dir++) {
+		if (!is_mounted_with_shared_option(*dir)) {
+			/* we need snap mount directory to become a mount point, so that we can
```

Comment:
The comment should start with a capital letter. While you are at it, the comment terminator marker can end on the 2nd line, after the dot that's missing there.